src/github3/orgs.py

```diff
@@ -1461,3 +1524,46 @@ def edit(self, state):
             self._update_attributes(json)
             return True
         return False
+
+
+class OrganizationHook(hook.Hook):
```

Comment:
We've experienced a significant instability in GitHub's API responses to the effect that we deliberately eliminated any object hierarchies like this. It probably seems a bit extreme, but I'd prefer to duplicate whatever needs duplicating from `repos.hook.Hook` here instead.

Reply:
Done